apps/dashboard/config/locales/en.yml

```diff
@@ -104,10 +105,8 @@ en:
       please ask your administrator to configure OnDemand to set the
       environment variable OOD_JOB_NAME_ILLEGAL_CHARS.
     batch_connect_sessions_delete_confirm: "Are you sure?"
-    batch_connect_sessions_delete_hover: "Delete Session"
     batch_connect_sessions_delete_title: "Delete"
     batch_connect_sessions_cancel_confirm: "Are you sure?"
-    batch_connect_sessions_cancel_hover: "Cancel Session"
```

Comment:
@adaybujeda FYI - I think I want to remove these 2 and just concatenate the title (`Cancel` and the word `Session`).